.github/workflows/create-pr.yml

```diff
@@ -0,0 +1,25 @@
+name: Pull Request Action
+on:
+  push:
+    branches:
+      - feat/bot-*
+jobs:
+  create-pull-request:
+    runs-on: ubuntu-latest
+    env:
```

Comment:
This should be given to the things that need it not too everything

Reply:
I think we have to define this GITHUB_TOKEN in env if we want to use gh commands.

Reply:
yes, but it should be defined on a per step not per job

Reply:
I see what you mean. Good point.